test/event_test.dart

```diff
@@ -1310,8 +1310,8 @@ void main() {
       final THUMBNAIL_BUFF = Uint8List.fromList([2]);
       Future<Uint8List> downloadCallback(Uri uri) async {
         return {
-          '/_matrix/media/v3/download/example.org/file': FILE_BUFF,
-          '/_matrix/media/v3/download/example.org/thumb': THUMBNAIL_BUFF,
+          '/_matrix/client/v1/media/download/example.org/file': FILE_BUFF,
```

Comment:
can we have the old ones and also add the new endpoints to make sure both are supported as expected?